amplipi/app.py

```diff
@@ -409,6 +412,14 @@ def announce(announcement: models.Announcement, ctrl: Api = Depends(get_ctrl)) -
   """ Make an announcement """
   return code_response(ctrl, ctrl.announce(announcement))
 
+# Internet radio catalog search
```

Comment:
I'm trying to figure out whether or not it makes more sense use a javascript library on the web app. This one looks like it would be pretty easy to integrate: https://github.com/nepodev/radio-browser.

Looking at moving to a js solution looks like it has some pros and cons:

Pros:
- removes an additional redirect for the station lookup
- keeps the API a little simpler

Cons:
- Doesn't support low powered interfaces 
- Additional js module to download


My instincts are to use the js solution assuming it is easy to integrate. What do you think?